src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/mock/WattTimeDataSourceMocker.cs

```diff
@@ -39,20 +41,31 @@ public void SetupDataMock(DateTimeOffset start, DateTimeOffset end, string locat
         {
             var newDataPoint = new GridEmissionDataPoint()
             {
-                BalancingAuthorityAbbreviation = defaultBalancingAuthority.Abbreviation,
                 PointTime = pointTime,
```

Comment:
This should also use the WattTimeTestData constants for Value, Version, Frequency, and Market